en/components/grids_templates/editing.md

```diff
@@ -288,6 +311,162 @@ These can be wired to user interactions, not necessarily related to the **@@igSe
+            if (!rowData) {
+                return;
+            }
+            if (event.newValue > rowData.UnitsInStock) {
+                event.cancel = true;
+                this.toast.show();
+            }
+        }
+    }
+}
+```
+If the value entered in a cell under the **Ordered** column is larger than the available amount (the value under **Units in Stock**), the editing will be cancelled and a toast with an error message will be displayed.
+}
+@@if (igxName === 'IgxTreeGrid') {
+```typescript
+export class MyTreeGridEventsComponent {
+    ...
+    public handleCellEdit(event: IGridEditEventArgs): void {
+        const row = this.data.find(e => e[this.grid.primaryKey] === event.cellID.rowID);
+        if (!row) {
+            return;
+        }
+        const column = this.grid.columnList.find(col => col.index === event.cellID.columnID);
+        if (column.field === "Age") {
+            if (event.newValue < 18) {
+                event.cancel = true;
+                this.toast.message = "Employees must be at least 18 years old!";
+                this.toast.show();
+            }
+        } else if (column.field === "HireDate") {
+            if (event.newValue > new Date().getTime()) {
+                event.cancel = true;
+                this.toast.message = "The employee hire date must be in the past!";
+                this.toast.show();
+            }
+        }
+    }
+}
+```
+Here, we are validating two columns. If the user tries to set an invalid value for an employee's **Age** (below 18) or their hire date (a future date), the editing will be cancelled (the value will not be submitted) and a toast with an error message will be displayed.
+}
+@@if (igxName === igxName === 'IgxHierarchicalGrid') {
+```typescript
+export class MyHGridEventsComponent {
+    ...
+    public handleCellEdit(event: IGridEditEventArgs, grid: IgxGridBaseComponent) {
+        const today = new Date();
+        const column = grid.columnList.find(col => col.index === event.cellID.columnID);
+        if (column.field === "Debut") {
+            if (event.newValue > today.getFullYear()) {
+                this.toast.message = "The debut date must be in the past!";
+                this.toast.show();
+                event.cancel = true;
+            }
+        } else if (column.field === "LaunchDate") {
+            if (event.newValue > new Date()) {
+                this.toast.message = "The launch date must be in the past!";
+                this.toast.show();
+                event.cancel = true;
+            }
+        }
+    }
+}
+```
+Since we're passing the grid as a second argument in the `handleCellEdit` calls, we're using the passed grid's columns to find to which column the cell belongs. With the above validation in place, when a user tries to change an artist's **Debut** year or an album's **Launch Date**, the grid will not allow any dates that are greater than today.
+}
+
+#### Editing Events - Demo
```

Comment:
_You don't need to split this section TBH. The entire section can be called "Events Configuration example" or something_

Reply:
It's convenient if you want to send a link to the demo - a user won't have to scroll through all of the configuration and can just lookup the `StackBlitz` example, in order to see if the implemented functionality suits their needs